core/App/fifo.cpp

```diff
@@ -47,7 +47,7 @@
 
 #define BUFFER_SIZE 1024
 
-#define UNIX_DOMAIN "/tmp/UNIX.domain"
+#define UNIX_DOMAIN "/var/run/dkey/dkeyprivision.sock"
```

Comment:
"/var/run/dkey/dkeyprivision.sock"  --> "/var/run/ehsm/dkeyprovision.sock"

Reply:
It will be change